### ISSUE TITLE
There is a version warning, is it possible to update the peer dependencies ?

### ISSUE THREAD

**Issue by zhanglolo** [iodigital-com/vite-plugin-msw#1]
``` sh
❯ pnpm add --save-dev @iodigital/vite-plugin-msw
Packages: +50
++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 374, reused 352, downloaded 2, added 50, done

devDependencies:
+ @iodigital/vite-plugin-msw 0.2.0

 WARN  Issues with peer dependencies found
.
└─┬ @iodigital/vite-plugin-msw 0.2.0
  └── ✕ unmet peer msw@^0.12.0: found 0.45.0
```

**Comment by zhanglolo** [comment]
I have a question, what's different bewteen using this plugin and follow office document [Getting Started](https://mswjs.io/docs/getting-started) ?
 Simplified steps of `npx msw init public/ --save` ? 

Looking forward to your reply

**Comment by zhanglolo** [comment]
@vanhoofmaarten i open PR for update dependencies. Would you have a look. thanks 🙏

**Comment by mrtnvh** [comment]
Thanks for your contributions. Will review and get back to you next week. Having some time off. ✌️

**Comment by mrtnvh** [comment]
Hi @zhanglolo, apologies for the delay. Updated all the packages and released v0.3.0. Could you try?
